lms/static/js/views/image_field.js

```diff
@@ -141,6 +141,7 @@
             },
 
             imageChangeSucceeded: function() {
+                this.$('.u-field-upload-button').css('opacity', '');
```

Comment:
Nit: I'd prefer that you add and remove a class to control the styling, rather than updating CSS attributes directly.